packages/mdc-ripple/README.md

```diff
@@ -46,7 +46,7 @@ In order to function correctly, MDC Ripple requires a _browser_ implementation o
 
 Because we rely on scoped, dynamic CSS variables, static pre-processors such as [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties) will not work as an adequate polyfill ([...yet?](https://github.com/postcss/postcss-custom-properties/issues/32)).
 
-Edge and Safari, although they do [support CSS variables](http://caniuse.com/#feat=css-variables), do not support MDC Ripple. See the respective caveats for [Edge](#caveat-edge) and [Safari](#caveat-safari) for an explanation.
+Edge and Safari < 10, although they do [support CSS variables](http://caniuse.com/#feat=css-variables), do not support MDC Ripple. See the respective caveats for [Edge](#caveat-edge) and [Safari < 10](#caveat-safari) for an explanation.
```

Comment:
Update this link to #caveat-safari--10 so it doesn't break with the updated heading

Reply:
Good catch! I added a backward compatible anchor tag (`<a name="caveat-safari"></a>`) to avoid breaking existing bookmarks.